instrumentation/apache-httpclient/apache-httpclient-5.0/src/main/java/io/opentelemetry/javaagent/instrumentation/apachehttpclient/v5_0/ResponsePayloadBridge.java

```diff
@@ -133,18 +133,18 @@ public static boolean isGzipped(Context context) {
 
   private static void checkForGzip(ResponsePayloadBridge bridge) {
     Optional<Header> contentEncodingHeader = Arrays.stream(bridge.headers)
-        .filter(header -> header.getName().equals("Content-Encoding"))
+        .filter(header -> header.getName().equalsIgnoreCase("content-encoding"))
         .findFirst();
-    if (contentEncodingHeader.isPresent() && contentEncodingHeader.get().getValue().equals("gzip")) {
+    if (contentEncodingHeader.isPresent() && "gzip".equalsIgnoreCase(contentEncodingHeader.get().getValue())) {
       bridge.isGzipped = true;
     }
   }
 
   private static void checkForChunked(ResponsePayloadBridge bridge) {
     Optional<Header> transferEncodingHeader = Arrays.stream(bridge.headers)
-        .filter(header -> header.getName().equals("Transfer-Encoding"))
+        .filter(header -> header.getName().equalsIgnoreCase("transfer-encoding"))
```

Comment:
The other direction `<constant>.equals[IgnoreCase](<nullable thing>)` is 100% better :-)

Reply:
Apologies, not sure why I didn't just do that yesterday